dist/stat_data.py

```diff
@@ -574,6 +564,7 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_set_ts_stable_upd', 'set timestamp stable updates'),
     TxnStat('txn_sync', 'transaction sync calls'),
     TxnStat('txn_timestamp_oldest_active_read', 'transaction read timestamp of the oldest active reader', 'no_clear,no_scale'),
+    TxnStat('txn_walk_concurrent_session', 'transaction walk of concurrent sessions'),
```

Comment:
@bdlucas1 do you think that a stat with the duration would be useful too?

Reply:
Yes, it would, particularly if this is something that multiple threads might be doing concurrently, as it would let us compute number of threads doing a walk, which is a direct indication of a bottleneck.

Reply:
I added another stat to track the total time spent on sessions walk.

Reply:
I removed the timing stat and kept the counter as it is. Will add it back if this approach shows some performance regressions.